app/models/user.js

```diff
@@ -1,41 +1,16 @@
 import Model, { belongsTo, attr } from '@ember-data/model';
-import { computed } from '@ember/object';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  firstName: attr('string'),
-  lastName: attr('string'),
-  emailLink: attr(),
-  phoneLink: attr(),
-  account: belongsTo('account'),
-  group: belongsTo('account-group', {
-    inverse: null,
-  }),
-  organization: belongsTo('organization'),
+export default class User extends Model {
+  @attr('string') firstName;
+  @attr('string') lastName;
+  @attr('email') email;
+  @attr('phone') phone;
 
-  email: computed('emailLink', {
-    get() {
-      return this.emailLink && this.emailLink.replace(/^mailto:/, '');
-    },
-    // eslint-disable-next-line no-unused-vars
-    set(key, value) {
-      this.set('emailLink', `mailto:${value}`);
-      return value;
-    },
-  }),
```

Comment:
Getting `user.email` changed in this file, the adding and removing of `mailto:` no longer happens
To my knowledge we never set the email or allowed it to be edited so the set portion is not needed.

Looking at DEV, this `mailto:` is actually in the data (first off, not sure how Hannes made a user account in DEV, acmidm shouldn't work on this one)
![image](https://user-images.githubusercontent.com/22245223/157633266-8f9ef9f4-7c32-40ab-a98f-107de76e9292.png)

quick check on PROD and the `mailto:` seems present in user data.

user.mail is used in these places:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/feature/KAS-3207-refactor-van-eenvoudige-ember-data-models-naar-octane/app/pods/settings/users/index/template.hbs#L83

https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/feature/KAS-3207-refactor-van-eenvoudige-ember-data-models-naar-octane/app/pods/settings/users/user/template.hbs#L26

Reply:
I will verify in a minute, but we're using the transforms `email` and `phone` for the `.email` and `.phone` attributes. These transforms come from this addon: https://github.com/mu-semtech/ember-mu-transform-helpers/ and they handle the stripping of `mailto:` and `tel:`

Reply:
Ah, I was not aware of this addon tbh.
Not completely sure if I like the approach of this addon, since you can't seem to choose which transforms you want.
We already have our own `date` and `datetime` transforms so which one wins?
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/feature/KAS-3207-refactor-van-eenvoudige-ember-data-models-naar-octane/app/transforms/date.js#L4

Also, it's hardly visible that those transforms get added to your code unless someone tells you about it...

Reply:
@sergiofenoll What you say is correct, the transforms are doing their job
the data is showing the uri with the prefixes
![image](https://user-images.githubusercontent.com/22245223/157642832-4e587ef2-1901-417d-9a0c-d86eebe801c0.png)

The displayed data does not have this prefixes
![image](https://user-images.githubusercontent.com/22245223/157642890-fcf8b4ae-4247-4097-97e6-aaec28c79919.png)